patchwise/main.py

```diff
@@ -19,11 +19,14 @@
 from .patch_review.ai_review.ai_review import add_ai_arguments, apply_ai_args
 from .patch_review.kernel_tree import create_git_worktree
 from .patch_review.patch_review import PATCH_PATH
+from .curse_setup import curse_pipeline
 
 logger = logging.getLogger(__name__)
 
 
 def parse_args() -> argparse.Namespace:
+    curse_pipeline()
```

Comment:
All logic related to curses for config should be done after parsing the config file and determining whether there's a need to prompt the user.
```suggestion
    # pseudocode
    if not config.do_not_show_again:
        curse_pipeline()
```